Tests/UnitTests/Networking/Responses/BaseHTTPResponseTest.swift

```diff
@@ -21,16 +21,31 @@ class BaseHTTPResponseTest: TestCase {
         file: StaticString = #filePath,
         line: UInt = #line
     ) throws -> T {
+        return try T.create(with: self.data(for: name, file: file, line: line))
+    }
+
+    @_disfavoredOverload
```

Comment:
This is a thing?! 🤯

Reply:
Yeah to resolve ambiguity for types that conform to both protocols :)